[Executorch][llama] Hookup use_attention_mask option in the source transforms inside llm mananger

kimishpatel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #10286
* #10285
* #10284
* #10283

Differential Revision: [D73222734](https://our.internmc.facebook.com/intern/diff/D73222734/)

kimishpatel committed:
[Executorch][llama] Hookup use_attention_mask option in the source transforms inside llm mananger



Differential Revision: [D73222734](https://our.internmc.facebook.com/intern/diff/D73222734/)

[ghstack-poisoned]

lucylq reviewed (approved):
Add a test to exercise this path?

kimishpatel committed:
Update on "[Executorch][llama] Hookup use_attention_mask option in the source transforms inside llm mananger"


Differential Revision: [D73222734](https://our.internmc.facebook.com/intern/diff/D73222734/)

[ghstack-poisoned]

kimishpatel committed:
Update on "[Executorch][llama] Hookup use_attention_mask option in the source transforms inside llm mananger"


Differential Revision: [D73222734](https://our.internmc.facebook.com/intern/diff/D73222734/)

[ghstack-poisoned]

kimishpatel committed:
Update on "[Executorch][llama] Hookup use_attention_mask option in the source transforms inside llm mananger"


Differential Revision: [D73222734](https://our.internmc.facebook.com/intern/diff/D73222734/)

[ghstack-poisoned]
